next/components/paragraph--accordion.tsx

```diff
@@ -0,0 +1,40 @@
+import * as AccordionUI from "@radix-ui/react-accordion";
+
+import { Paragraph } from "@/components/paragraph";
+import { Accordion } from "@/lib/zod/paragraph";
+import ChevronIcon from "@/styles/icons/chevron.svg";
+
+export function ParagraphAccordion({ paragraph }: { paragraph: Accordion }) {
+  return (
+    <div className="relative mb-7 h-full rounded border bg-white p-4 transition-all hover:shadow-md">
+      <h2 className="text-heading-sm font-bold md:text-heading-md">
+        {paragraph.field_heading}
+      </h2>
+      <AccordionUI.Root type="single" collapsible>
+        {paragraph.field_accordion_items &&
+          paragraph.field_accordion_items.map((item) => (
+            <AccordionUI.Item
+              className="rounded border"
+              key={item.id}
+              value={item.id}
+            >
+              <AccordionUI.Header className="bg-wunderpurple-400 p-2 text-md text-white md:text-lg">
+                <AccordionUI.Trigger className="group flex w-full flex-row items-center gap-1 md:gap-1.5">
+                  <ChevronIcon
+                    aria-hidden
+                    className="h-3 w-3 group-data-[state=open]:rotate-180 md:h-7 md:w-6"
```

Comment:
I had no idea you could do this, assign a group property to the parent, then add a class for an attribute selector of the parent.. cool.

![image](https://user-images.githubusercontent.com/185412/222533100-567b48cd-7f6a-4b37-a2a2-89d4218158c6.png)

Reply:
Super handy!